lb-haproxy-study/LB-HAProxy-ds-sc/run_me.sh

```diff
@@ -52,34 +52,34 @@ vagrant ssh master1 -- sed -i 's/CHANGE_CERT_KEY/'"$KEY"'/g' /home/vagrant/kubea
 echo "KUBEADM INIT"
 
 vagrant ssh master1 -- sudo kubeadm init --upload-certs --config=/home/vagrant/kubeadm-config.yaml | tee init_results.txt
-cat init_results.txt  | grep 'To start using your cluster' -A 4 | tail -n 3 > masters_kubeconfig.sh
+grep 'To start using your cluster' -A 4 < init_results.txt | tail -n 3 > masters_kubeconfig.sh
```

Comment:
This should work, but it would work with `grep 'To start using your cluster' init_results.txt' | ...` as well.

I'd prefer if you'd move the `-A 4` next to grep though, ie. `grep -A4 ...`.